src/Label/Commands/Create.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace CultuurNet\UDB3\Label\Commands;
+
+use CultuurNet\UDB3\Label\ValueObjects\Privacy;
+use CultuurNet\UDB3\Label\ValueObjects\Visibility;
+use ValueObjects\Identity\UUID;
+use ValueObjects\String\String as StringLiteral;
+
+class Create extends AbstractCommand
```

Comment:
Create lijkt me nogal algemeen. Het staat wel in de label namespace. De vraag is wat we nu doen qua naamgeving? CreateLabel, ook al staat het in de label namespace? Of enkel de actie Create, aangezien het in de label namespace staat?